tests/python/pants_test/backend/jvm/tasks/jvm_compile/zinc/zinc_compile_integration_base.py

```diff
@@ -38,6 +38,31 @@ def test_scala_compile_jar(self):
         self.assertTrue(jar.getinfo(SHAPELESS_CLSFILE),
                         'Expected a jar containing the expected class.')
 
+  # TODO: this could be converted into a unit test!
+  def test_scala_profile_can_be_generated(self):
+    with temporary_dir() as tmp_dir:
```

Comment:
It's not immediately clear from the title why this would not be the case. I think it would be helpful to link to this PR's description with a brief note explaining why we have to test this case.

Reply:
Will do!